### PR TITLE
[stable8.2] backport of #23728

### PR DESCRIPTION
**Pull request by michaelstingl**
Sanitize more data in config report

@PVince81 Here is my proposal to fix owncloud/enterprise#1470 . Could you check? /cc @cdamken

### CODE REVIEW

**Comment by PVince81** [comment]
moved to 8.2.8 since we're past code freeze

**Comment by cdamken** [comment]
:+1:

File: lib/private/systemconfig.php [hunk 1]
```diff
@@ -41,7 +41,19 @@ class SystemConfig {
...
+		'log.condition' => [
+			'shared_secret' => true,
+		],
+		'license-key' => true,
+		'redis' => [
+			'password' => true,
+		],
+		'objectstore' => [
+			'arguments' => [
+				'password' => true,
```
Comment (VicDeo) [inline lib/private/systemconfig.php]: Would  this work? As I see  `removeSensitiveValue` is not capable to work with the third level so the whole `arguments` array will be stripped

**Comment by michaelstingl** [comment]
@VicDeo Does this mean backport isn't possible? /cc @felixboehm

**Comment by VicDeo** [comment]
@michaelstingl no, this means that https://github.com/owncloud/core/pull/23728 introduces a bug ;)

**Comment by VicDeo** [comment]
@michaelstingl  ah, there is a recursion in `removeSensitiveValue` 
sorry for confusion.

make sense :+1:

**Pull request merged**